### DISCUSSION TITLE
API Infrastructure

### DISCUSSION THREAD

**Discussion by cweinschenk** [ulfsri/neris-framework#17] [Announcements]
Let's use this discussion as the thread for API related content sharing and discussion.

The NERIS API documentation can be found here: [Swagger Docs](https://api-test.neris.fsri.org/v1/docs#/Incidents/validate_incident_incident__neris_id_entity__validate_post)

For those who are interested, a NERIS API client was built in python can be found here: [pypi package](https://pypi.org/project/neris-api-client/) with the source code here: [API Client Repo](https://github.com/ulfsri/neris-api-client)

To request access to sandback, please use this link: [NERIS Helpdesk](https://neris.atlassian.net/servicedesk/customer/portal/3/group/3/create/10027)

**Comment by bbannach-fp** [comment]
I submitted a request for access last week Monday, do you know the typical response time for getting access?  I still have not heard back, nor has anyone on my team.

**Reply by cweinschenk** [reply]
Hi @bbannach-fp -- we apologize for the delay. The initial plan was to get the invites out Monday/Tuesday time frame, but we a running a couple days behind at the moment but invites will be out this week. We appreciate your patience.

**Reply by essengee** [reply]
Hi @cweinschenk we have received logins but at the moment it seems to just be a single webpage with a link back to the API documentation that we already have access to. Is there more documentation somewhere on how to actually connect to the sandbox - acquire tokens etc?

**Reply by cweinschenk** [reply]
Hi @essengee we just posted a discussion with more information that should help with the sandbox: [API Info](https://github.com/ulfsri/neris-framework/discussions/18)

**Reply by essengee** [reply]
@cweinschenk Thank you!
